components/style/media_queries.rs

```diff
@@ -146,6 +146,7 @@ impl ToCss for MediaQuery {
             let (mm, l) = match e {
                 Expression::Width(Range::Min(ref l)) => ("min", l),
                 Expression::Width(Range::Max(ref l)) => ("max", l),
+                Expression::Width(Range::Eq(ref l)) => ("eq", l),
```

Comment:
This would serialize the query as `eq-width: foo`, which isn't what is supposed to happen (it should be serialized just as `width: foo`.

Please fix it and write a test for it :)